Sources/RichTextKit/Coordinator/RichTextUserInitiatedAction.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  RichTextUserAction.swift
+//
+//
+//  Created by Dominik Bucher on 07.12.2023.
+//
+
+import Foundation
+
+enum RichTextUserInitiatedAction {
+    case backgroundColor(ColorRepresentable)
+    case foregroundColor(ColorRepresentable)
+    case highlightedRange(NSRange?)
+    case highlightingStyle(RichTextHighlightingStyle)
+    case shouldPasteImage(Insertion<ImageRepresentable>)
+    case shouldPasteImages(Insertion<[ImageRepresentable]>)
+    // TODO: Paste attributed string??
```

Comment:
Should this TODO be solved/removed?

Reply:
Paste attributed string is in the making in other branch (local) and our repo. Deleted TODO, will do later as separate PR.